client/state/sites/selectors/is-simple-seller.ts

```diff
@@ -0,0 +1,24 @@
+import 'calypso/state/ui/init';
+import { getSiteOption, getSitePlanSlug } from 'calypso/state/sites/selectors';
+import type { AppState } from 'calypso/types';
+
+/**
+ * Returns true if the current site is a simple seller site (payment block, no Woo plan)
+ *
+ * @param {object} state Global state tree
+ * @returns {boolean}
+ */
+export default function isSimpleSeller( state: AppState, siteId: number ): boolean | null {
+	// Check if the site intent is 'sell'.
+	const intent = getSiteOption( state, siteId, 'site_intent' );
+	if ( ! intent ) {
+		return null;
```

Comment:
I think this should return false as well, for consistency.

Reply:
I was conflicted about that but I think you're right. Fixed!